connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -429,9 +431,13 @@ private void convertMessages(ConsumerRecords<byte[], byte[]> msgs) {
                     msg.offset(),
                     ConnectUtils.checkAndConvertTimestamp(msg.timestamp()),
                     msg.timestampType());
+            log.trace("{} applying transformations to message with key {}, value {}", this, msg.key(), msg.value());
```

Comment:
I'm not sure we get a useful `toString` from `byte[]`. Maybe it would be more helpful to mention the topic, partition, and offset of the message instead?